src/components/content/medications-table-base.tsx

```diff
@@ -36,6 +36,30 @@ export const MedicationsTableBase = ({
         </>
       ),
     },
+    {
+      title: "Dispensed",
+      render: (medication: MedicationStatementModel) => (
```

Comment:
Don't need to do this now, but realized today myself that we don't need types here:
```suggestion
      render: (medication) => (
```